README.md

```diff
@@ -149,14 +151,7 @@ Install the package using:
 
     sudo apt-get install sqlitebrowser
 
-Ubuntu 14.04.X, 15.04.X, 15.10.X and 16.04.X are supported for now (until
-Launchpad decides to discontinue building for any series).
-
-Ubuntu Precise (12.04) and Utopic (14.10) are not supported:
-* Precise does not have a new enough Qt package in its repository by default,
-  which is a dependency
-* Launchpad does not support Utopic any more, which has reached its End of
-  Life
+Packages for Older Ubuntu releases are supported until launchpad keeps building those or if Older Ubuntu release has dependency packages that are required to build the latest version of Sqlitebrowser. We don't remove builds from our ppa repos, so users can still install older version of sqlitebrowser if they like. Alterntaively Linux users can also switch to Snap packages if Snap packages are supported by the distro they are using.
```

Comment:
Should this be "while launchpad keeps building those..." rather than "until launchpad keeps building those..."?

Also there's a typo in 'Alternatively'. :wink:

Reply:
Thanks, seems like morning coffee issue. :smile: